packages/backend/src/runtime.ts

```diff
@@ -11,12 +11,18 @@
  *
  * TODO: Support TS runtime modules
  */
-
-// @ts-ignore - These are package subpaths
-import crypto from '#crypto';
+let webcrypto;
```

Comment:
Why did you remove the subpath import here? Are you trying to make Jest work without configuring a `moduleMapper`? If so, Jest claims that that subpath imports work since this release: https://github.com/jestjs/jest/releases/tag/v29.4.0 :) #til 

I haven't tested the following, but this is what I had in mind for `#crypto`: 

`node/crypto.mjs`:
```
export { webcrypto } from 'node:crypto';
```

`node/crypto.js`:
```
module.exports.webcrypto = require('node:crypto').webcrypto;
```

`package.json`:
```
  "imports": {
    "#crypto": {
      "edge-light": "./dist/runtime/browser/crypto.mjs",
      "worker": "./dist/runtime/browser/crypto.mjs",
      "browser": "./dist/runtime/browser/crypto.mjs",
      "node": {
        "require": "./dist/runtime/node/crypto.js",
        "import": "./dist/runtime/node/crypto.mjs"
      },
      "default": "./dist/runtime/browser/crypto.mjs"
    }
  },
```

`runtime.ts`:
```
import { webcrypto } from '#crypto';
...
```

Reply:
Change reverted and suggestion applied.